vispy/scene/cameras/cameras.py

```diff
@@ -258,32 +518,75 @@ def rect(self, args):
         ViewBox. 
         """
         if isinstance(args, tuple):
-            self._rect = Rect(*args)
+            rect = Rect(*args)
         else:
-            self._rect = Rect(args)
-        self._update_transform()
-
-    @property 
-    def invert_y(self):
```

Comment:
Not anymore. Now we have `aspect_ratio`. Just do `cam.aspect_ratio = 1, -1` to flip the y-axis.